apps/extension/src/routes/popup/home/index.tsx

```diff
@@ -14,22 +13,8 @@ export interface PopupLoaderData {
 // Because Zustand initializes default empty (prior to persisted storage synced),
 // We need to manually check storage for accounts & password in the loader.
 // Will redirect to onboarding or password check if necessary.
-export const popupIndexLoader = async (): Promise<Response | PopupLoaderData> => {
-  const wallets = await localExtStorage.get('wallets');
-
-  if (!wallets.length) {
-    await chrome.tabs.create({ url: chrome.runtime.getURL(`page.html`) });
-    window.close();
-  }
```

Comment:
Are we OK with not checking for wallets anymore? Don't we want to check `await needsLogin()` _and_ `await needsOnboard()`?

Reply:
the login page cascades to needsOnboard